crates/wasmtime/src/runtime/vm/traphandlers.rs

```diff
@@ -668,8 +669,11 @@ impl CallThreadState {
         faulting_addr: Option<usize>,
         call_handler: impl Fn(&SignalHandler) -> bool,
     ) -> TrapTest {
+        log::trace!("testing if trap: {faulting_addr:?}, {regs:?}");
```

Comment:
I think we'll want to remove all the logging here because this is called from a signal handler. Or maybe a per-module debugging macro which can be toggle locally without committing it or something like that?